sentry/templates/configmap-relay.yaml

```diff
@@ -37,5 +37,8 @@ data:
       {{- else }}
       redis: "redis://{{ $redisHost }}:{{ $redisPort }}"
       {{- end }}
+      topics:
```

Comment:
I'm curios whether those topics expected to be handled by `auto.create.topics.enable` property or actually created by sentry migration/init job?

Reply:
Interesting question
It appears that only place that creates topics in advance is Snuba - uses this list here https://github.com/getsentry/snuba/blob/22.11.0/snuba/utils/streams/topics.py#L6

This doesn't go through snuba., It is relay -> sentry. It is the same as ingest-consumer (for example `ingest-transactions` topic) - these are created using `auto.create.topics.enable` (on by default because of https://github.com/getsentry/sentry/blob/22.11.0/src/sentry/conf/server.py#L2578)